### ISSUE TITLE
something wrong

### ISSUE THREAD

**Issue by 1jbr** [LagoLunatic/DSVEdit#57]
![image](https://user-images.githubusercontent.com/92062650/136297534-7a21cf61-f70e-4d9f-8a41-a50555eb0828.png)

this quickly appears when i open DSVEdit.exe, version.rb and dsvedit.rb. is this normal?

**Comment by LagoLunatic** [comment]
No, it's not normal. However your screenshot doesn't include the important part of the error message (the top) so I've got no idea what's wrong.

**Comment by 1jbr** [comment]
you know how hard is to screenshot a thing that literally appears in a fraction of a second

**Comment by piratesephiroth** [comment]
@1jbr 

open `lib\ruby\2.4.0\win32\registry.rb` in a text editor

scroll all the way down to end of the file and change
```ruby
str.encode(enc)
```
to
```ruby
str.encode("utf-8")
```

see if it works after that

**Comment by LagoLunatic** [comment]
I'd be careful editing ruby's source for stuff involving the registry, wouldn't want to corrupt your registry. I really doubt that's the correct solution, but if you do try to edit it make a backup of your registry beforehand.

**Comment by piratesephiroth** [comment]
Well, I'm hoping that will be edited in the Ruby library that comes with DSVania Editor so it's probably harmless (dsvedit doesn't write anything to the registry, does it?).

I think that error happens when it's enumerating the recently opened files or something.

**Comment by piratesephiroth** [comment]
Nah that should be a bug in Ruby. I've seen it happening in a non-japanese system.
Probably some recently opened file with a name that's not being converted properly.
